fastlane/Pluginfile

```diff
@@ -7,8 +7,8 @@ group :screenshots, optional: true do
 end
 
 # This comment avoids typing to switch to a development version for testing.
-# gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit', branch: 'trunk'
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 2.3'
+gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit', branch: 'trunk'
+# gem 'fastlane-plugin-wpmreleasetoolkit', '~> 2.3'
```

Comment:
Before merging this (or #17891) we ought to ship a new version of the release toolkit.

Reply:
v3.0.0 (https://github.com/wordpress-mobile/release-toolkit/pull/334) is almost ready to go

Reply:
Let's land this, then update #17891 to use the new version of the toolkit.

Reply:
SGTM 👍